README.md

```diff
@@ -20,16 +20,19 @@ var after = new mapboxgl.Map({
   style: 'mapbox://styles/mapbox/dark-v9'
 });
 
-new mapboxgl.Compare(before, after, {
+// A selector or reference to HTML element
+var container = '#comparison-container';
+
+new mapboxgl.Compare(before, after, container, {
   mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
-  orientation: 'vertical' // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical
+  orientation: 'vertical', // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical,
 });
 ```
 
 ### Methods
 
 ```js
-compare = new mapboxgl.Compare(before, after, {
+compare = new mapboxgl.Compare(before, after, container, {
```

Comment:
so this would be a breaking API change. We'd need to make that clear in the CHANGELOG and provide an upgrade guide.

Reply:
Thanks for calling that out @andrewharvey! I had planned on adding a changelog entry when releasing the next package version, but highlighting that necessity is good 👍